deepchem/molnet/load_function/material_datasets/load_bandgap.py

```diff
@@ -112,91 +93,13 @@ def load_bandgap(
   Examples
   --------
   >> import deepchem as dc
-  >> tasks, datasets, transformers = dc.molnet.load_bandgap(reload=False)
+  >> tasks, datasets, transformers = dc.molnet.load_bandgap()
   >> train_dataset, val_dataset, test_dataset = datasets
   >> n_tasks = len(tasks)
   >> n_features = train_dataset.get_data_shape()[0]
   >> model = dc.models.MultitaskRegressor(n_tasks, n_features)
```

Comment:
Is it possible to use `>>>` now so that these examples render correctly in the docs and are checked with doctest?

Reply:
I don't think we can have doctest running these examples, because that would make it way too slow.

Reply:
@nd-02110114 Suggested a trick for the rendering:

```
>>> ...
>> tasks, datasets, transformers = dc.molnet.load_bandgap()
>> ...
```

That is, you have the first line with a `>>>` to render it correctly, but you don't actually run the code. I think there are a few examples of this elsewhere in DeepChem

Reply:
I'll give it a try!

Reply:
Fixed!